src/library/Link/styled.js

```diff
@@ -26,7 +26,7 @@ export const Link = styled('a', {
 
     '&:hover': {
       color: theme.Link_color_hover,
-      textDecoration: 'underline'
+      ...(underline && { textDecoration: 'underline' })
```

Comment:
Assuming change above, you can do this here:
```es6
textDecoration: textDecoration || 'underline',
```
Which will support the default behavior as well as the custom one.